tests/test_cli_socket_autodiscovery.py

```
@@ -47,19 +57,38 @@ def wait_ready(self, timeout: float) -> bool:
+    def stop(self) -> None:
+        deadline = time.monotonic() + 2.0
+        while self._thread.is_alive() and time.monotonic() < deadline:
+            try:
+                client = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+                client.settimeout(0.2)
+                client.connect(self.socket_path)
+                client.sendall(b"ping\n")
+                try:
+                    client.recv(1024)
+                except OSError:
+                    pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)